src/components/Game.js

```diff
@@ -1,39 +1,79 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import Analyzer from './Analyzer.js';
+import Notes from './NoteDetails.js';
+import Info from './Info.js';
 
 class Game extends Component {
   constructor(props){
     super(props);
     this.state={
-      // last_propmted_note: null,
-      // current_prompted_note: null,
-      currentUserNote: null,
+      lastPromptedNoteNum: null,
+      promptedNoteNum: null,
+      promptedNoteLetter: null,
+      promptedNoteFreq: null,
+      userPlayingNote: null,
       // note_matched: false
+      infoShown: false
     }
+  this.toggleInfoShown = this.toggleInfoShown.bind(this);
+  this.increaseSkippedCountCallback = props.increaseSkippedCountCallback.bind(this);
+  }
 
+  toggleInfoShown () {
+    this.setState({ infoShown: !this.state.infoShown });
   }
 
-  getCurrentUserNote = (note) => {
-    this.setState({currentUserNote: note})
-    console.log('in game, currentUserNote: ', this.state.currentUserNote);
+  componentDidMount() {
+    this.setNewNote();
   }
 
-  componentWillUnmount() {
-    if (this.state.processor){this.state.processor.disconnect()};
+  setNewNote() {
+    let noteNum = null;
+    let noteDetails = null;
+    while(noteNum === null || noteDetails === undefined || noteNum === this.state.lastPromptedNoteNum) {
+      noteNum = this.getRandomIntInclusive(16, 42);
+      console.log('noteNum in setNewNote: ', noteNum);
+      noteDetails = this.getNoteDetails(noteNum);
+    }
+
+    this.setState({
+      promptedNoteNum: noteNum,
+      promptedNoteLetter: noteDetails.noteName,
+      promptedNoteFreq: noteDetails.frequency
+    });
+
+    console.log('promptedNoteNum: ', this.state.promptedNoteNum, 'promptedNoteFreq: ', this.state.promptedNoteFreq, 'promptedNoteLetter: ', this.state.promptedNoteLetter);
+  }
+
+  getRandomIntInclusive(min, max) {
+      const randInt = Math.floor(Math.random() * (max - min + 1)) + min;
+      console.log('randInt in getRandomIntInclusive: ', randInt);
+      return randInt;
   }
 
+  getNoteDetails(noteNum) {
+    let note = Notes[noteNum];
+    console.log('note: ', note);
+    return note;
+  }
+
+  getUserPlayingNote = (note) => {
+    this.setState({userPlayingNote: note})
+    // console.log('in game, userPlayingNote: ', this.state.userPlayingNote);
+  }
 
-  showMeTheState = () => {
-    // console.log('in showMeTheState, userAudioFromMic: ', this.state.userAudioFromMic);
-    // console.log('in showMeTheState, userAudioFromMic type: ', typeof this.state.userAudioFromMic);
+  componentWillUnmount() {
+    if (this.state.processor){this.state.processor.disconnect()};
+  }
 
-    // const numOfMics = this.state.userAudioFromMic.mediaStream.getAudioTracks();
-    // console.log('numOfMics: ', numOfMics);
+  skipNote = () => {
+    this.increaseSkippedCountCallback();
```

Comment:
skipping a note will not prevent this note from appearing again, is this intended?

Reply:
Yes, this is intended. After talking with other musicians, I decided to leave it like this for now. We can add a game option to "play every note" later. As a baseline game, it makes sense to have repetition for beginner learners.